spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+describe Api::V1::UsersController do
+  describe "editing a user" do
+    it "can add a cohort to a user" do
+      user = create(:user, cohort_id: nil)
+      cohort = create(:cohort)
+
+      put :update, format: :json,
+                   id: user.id,
+                   user: {
+                            cohort_id: cohort.id
+                          }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.